src/features/conversion.ts

```diff
@@ -270,7 +270,7 @@ export function createLinearScaling(
             }
 
             return Decimal.sub(baseAmount, unref(processedBase))
-                .sub(1)
+                // .sub(1)
```

Comment:
Needs to be removed to make `currentGain` correct, but it throws the others out of wack

Reply:
It didn't, I just didn't math right